rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1.json

```diff
@@ -131,6 +131,12 @@
       "description": "Document is harvested or not, will disable record edition or similar.",
       "type": "boolean",
       "default": false
+    },
+    "_draft": {
+      "title": "Draft",
+      "description": "Document is a draft record or not, will disable record validation.",
+      "type": "boolean",
+      "default": false
```

Comment:
I think this field doesn't need to be displayed in the editor ? If not, then you need to specify to editor to hide it : 
```
"_draft": {
    "title": "Draft",
    "description": "Document is a draft record or not, will disable record validation.",
    "type": "boolean",
    "default": false
    "form": {
        "hideExpression": "true"
    }
}
```